src/mpi/romio/adio/ad_gpfs/bg/ad_bg_pset.c

```diff
@@ -16,14 +16,14 @@
 // #define bridgeringaggtrace 1
 
 #include <stdlib.h>
+#include <stdbool.h>
```

Comment:
Okay, I think that's fine. It shouldn't be hard to scrub the entire repo for `bool` usage once we get it right. Alternatively we may just assume the availability of `bool` the way we did for fixed-width integers a while back.